docs/getting-started.md

```diff
@@ -35,5 +48,5 @@ curl https://raw.githubusercontent.com/kuadrant/multicluster-gateway-controller/
 
 ### What's Next
 
-* [Multicluster Gateways walkthrough](how-to/multicluster-gateways-walkthrough.md)
-* [Kuadrant Documentation](https://docs.kuadrant.io).
+Now that you have two Kind clusters configured with the Multicluster Gateway Controller installed you are ready to begin [the Multicluster Gateways walkthrough.](how-to/multicluster-gateways-walkthrough.md)
```

Comment:
`the Multicluster Gateway Controller` -> `Kuadrant`

Reply:
ill update the links to